controllers/forum.js

```diff
@@ -0,0 +1,77 @@
+const   express     = require('express'),
+        Forum       = require('../models/forum'),
+        router      = express.Router();
+
+        //FIND ALL
+        router.get('/', async (request,response)=>{            
+            await Forum.find({},(error,allForums)=>{
+                if(error)
+                {
+                    console.log(error);
+                    response.redirect('/');
+                }
+                else
+                {
+                    response.send(allForums);
+                }
+            });
+        });
+
+        //CREATE 
+        router.post('/', async (request,response)=>{            
+             await Forum.create(request.body,(error,newForum)=>{
+                if(error)
+                {
+                    console.log(error);
+                    response.redirect('/api/forum');
```

Comment:
Also, in case of error, don't redirect. Send them a 500 or something. Write good error messages and statuses.

https://www.restapitutorial.com/httpstatuscodes.html